### PR TITLE
Update compute tutorial

### PR DESCRIPTION
**Pull request by dongreenberg**

### CODE REVIEW

**Comment by dongreenberg** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #347** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/347?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/run-house/runhouse/347?utm_source=stack-comment).

File: docs/tutorials/api/compute.rst [hunk 1]
```diff
@@ -37,79 +36,43 @@ Optionally, login to Runhouse to sync any secrets.
...
-   already have up, and access through an IP address and SSH
-   credentials. Please refer to the :ref:`Bring-Your-Own Cluster` section for further instructions.
-
-2. **On-demand Cluster**: Associated with your cloud account (AWS, GCP,
-   Azure, LambdaLabs). There are additional features for these clusters,
-   such as cluster (auto) stop. Please refer to
-   :ref:`On-Demand Cluster` for instructions on first getting
-   cloud credentials set up.
-
-3. **SageMaker Cluster (Alpha)**: Clusters that are created and managed
```
Comment (carolineechen) [inline docs/tutorials/api/compute.rst:59]: just checking we want to remove sagemaker from this? this tutorial is still currently acting as the main compute tutorial until we split them up

File: docs/tutorials/api/compute.rst [hunk 2]
```diff
@@ -268,293 +158,68 @@ and the function automatically runs on your specified hardware!
...
-    INFO | 2023-08-29 04:00:01.870718 | Setting up Function on cluster.
-    INFO | 2023-08-29 04:00:01.873021 | Copying package from file:///Users/caroline/Documents/runhouse/runhouse to: cpu-cluster
-    INFO | 2023-08-29 04:00:03.145979 | Copying package from file:///Users/caroline/Documents/runhouse/runhouse to: cpu-cluster
-    INFO | 2023-08-29 04:00:04.625905 | Calling env_20230829_035957.install
-
-
-.. parsed-literal::
-    :class: code-output
-
-    base servlet: Calling method install on module env_20230829_035957
```
Comment (carolineechen) [inline docs/tutorials/api/compute.rst:330]: I cleared the output for a bunch of these longer log outputs. I think we also need to revisit which ones we set to be logger.info vs debug in terms of what processes we want to be logged during processes. and possibly in tutorials keep it at the info/error level for cleanliness if we want to show output.

Comment (dongreenberg) [reply docs/tutorials/api/compute.rst:330]: Oh I thought I cleared all output before I pushed. Yes, makes sense.

Comment (carolineechen) [reply docs/tutorials/api/compute.rst:330]: yup you did clear them -- I generally like to leave them in to display the output of the code (for users that might just want to skim through without setting up yet), but removed the more cluttery setup/logs output

**Review by carolineechen — APPROVED** [review]

**Pull request merged**